agent/test-agent-cli/design/RUNBOOK.md

```diff
@@ -0,0 +1,195 @@
+# Steps to use Test-agent Command Line Interface
```

Comment:
Overall: Good work, can you run through each step and make them more command based, i.e 
1: Install kind (kind link)
2: Create a kind cluster that will be used for testsys: `kind create cluster --name agent-cli` 
...

Reply:
Also make sure that the steps can be run through verbatim.

Reply:
I tried to make commands more direct now :)